src/toggl_api.cc

```diff
@@ -1496,3 +1496,8 @@ void toggl_iam_click(void *context,
     }
     app(context)->TrackInAppMessage(type);
 }
+
+char_t *toggl_format_duration_time(void *context,
```

Comment:
Basically, during the resize session, the TE doesn't save to the Library yet. So I Intentionally expose this format duration to the app, since I need a consistent way to format the Duration.

<img width="147" alt="Screen Shot 2020-02-04 at 14 15 29" src="https://user-images.githubusercontent.com/5878421/73722366-e604f480-4758-11ea-8904-4d5adf5ba95d.png">

Reply:
👍